src/components/Layout/index.tsx

```diff
@@ -10,7 +10,7 @@ import {
   RIGHT_NAV_LINKS,
   SOCIAL_LINKS
 } from '../../utils/globalnav';
-import React from 'react';
+import React, { useEffect, useState } from 'react';
```

Comment:
this isn't entirely necessary to change but the `React` import is no longer required for React 17+ https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html